cypress/integration/happy-path-with-accessibility.js

```diff
@@ -110,24 +109,55 @@ describe('Get help to retrain smoke test', function() {
   });
 
   it('should allow me to choose training options', function() {
+    cy.get('[id="training_english_skills"]').check()
+    cy.get('[id="training_math_skills"]').check()
     cy.contains('Continue').click();
     checkAccessibility();
   });
 
-  it('should allow me to choose computer skills training options', function() {
+  it('should allow me to choose job hunting advice', function() {
+    cy.get('[id="it_training_computer_skills"]').check()   
     cy.contains('Continue').click();
     checkAccessibility();
   });
 
-  it('should allow me to choose job hunting advice and navigate to action plan', function() {
+  it('should allow me to find jobs near me and navigate to action plan', function() {
+    cy.get('[id="job_hunting_cv"]').check();
+    cy.get('[id="job_hunting_cover_letter"').check();
+    cy.get('[id="job_hunting_interviews"]').check(); 
     cy.contains('Continue').click();
+    cy.contains('Show jobs near me').click();
+    checkAccessibility();
+  })
+
+  it('should allow me to find an maths course', function() {
+    cy.go('back')
+    cy.contains('Find a maths course').click()
     checkAccessibility();
   });
 
-  it('should allow me to find jobs near me', function() {
-    cy.contains('Show jobs near me').click();
+  it('should allow me to find an English course', function() {
+    cy.go('back')
+    cy.contains('Find an English course').click()
     checkAccessibility();
-  })
+  });
+
+  it('should allow me to find local schemes and offers', function() {
+    cy.go('back')
+    cy.contains('Show me local offers').click()
+    checkAccessibility();
+  });
+
+  it('should allow me to save my results', function() {
+    cy.get('a[href="/save-your-results"]').click()
```

Comment:
something we need to think about is return to saved results and how to test that, that's a different journey with different pages. Access to that will happen before having selected a job and won't allow us to do this here, how can we integrate it to our accessibility test? Maybe we need 2 different tests to cover save and return separately rather than have save your results here

Reply:
Agreed, probably best to look into this again as journey has changed but leave one journey where the user does save results after choosing a job profile makes sense to cover the vanilla scenario.

Reply:
after you address the previous comments let's discuss in person, as we should also cover the error pages

Reply:
Agreed. Will work on this on a separate PR.